src/checkstatus/SSMAutomationExecutionStatusWaiter.py

```diff
@@ -0,0 +1,31 @@
+import botocore
+import constants
+import waiter_acceptor_config
+
+def check_automation_execution_status(automation_execution_id, max_attempts, ssm):
+    acceptor_config = waiter_acceptor_config.SSM_AUTOMATION_EXECUTION_COMPLETED
+    model = botocore.waiter.WaiterModel({
```

Comment:
Can we define the waiter model globally as a constant outside if the method, which should load the waiter into memory once and can be reused in subsequent invocations

Reply:
do you mean add it in the caller function?

Reply:
i meant outside of the function scope

Reply:
its defined within the method because the max attempts value is different on every retry

Reply:
ok